dags/inference/maxtext_inference.py

```diff
@@ -16,204 +16,490 @@
-          ]
-          model_configs["moe_matmul"] = sweep_model_configs.get(
-              "moe_matmul", "false"
-          )
-          if tpu_version == TpuVersion.V5E:
-            # v5e benchmarks
-            project_name = Project.TPU_PROD_ENV_AUTOMATED.value
-            zone = Zone.US_EAST1_C.value
-            network = V5_NETWORKS
-            subnetwork = V5E_SUBNETWORKS
-            runtime_version = RuntimeVersion.V2_ALPHA_TPUV5_LITE.value
-          elif tpu_version == TpuVersion.V5P:
-            zone = Zone.US_EAST5_A.value
-            runtime_version = RuntimeVersion.V2_ALPHA_TPUV5.value
-            project_name = Project.TPU_PROD_ENV_AUTOMATED.value
-            network = V5_NETWORKS
-            subnetwork = V5P_SUBNETWORKS
-
-          maxtext_stable_1slice = maxtext_inference_gce_config.get_maxtext_inference_nightly_config(
-              tpu_version=tpu_version,
-              tpu_cores=tpu_cores,
-              tpu_zone=zone,
-              runtime_version=runtime_version,
-              project_name=project_name,
-              time_out_in_min=60,
-              is_tpu_reserved=True,
-              test_name=f"{test_name_prefix}-stable-{model}-per_device_batch_size-{per_device_batch_size}-ici-fsdp{ici_fsdp}-ar{ici_ar}-tensor{ici_tensor}",
-              test_mode=SetupMode.STABLE,
-              network=network,
-              subnetwork=subnetwork,
-              model_configs=model_configs,
-          )
-          maxtext_nightly_1slice = maxtext_inference_gce_config.get_maxtext_inference_nightly_config(
-              tpu_version=tpu_version,
-              tpu_cores=tpu_cores,
-              tpu_zone=zone,
-              runtime_version=runtime_version,
-              project_name=project_name,
-              time_out_in_min=60,
-              is_tpu_reserved=True,
-              test_name=f"{test_name_prefix}-nightly-{model}-per_device_batch_size-{per_device_batch_size}-ici-fsdp{ici_fsdp}-ar{ici_ar}-tensor{ici_tensor}",
-              test_mode=SetupMode.NIGHTLY,
-              network=network,
-              subnetwork=subnetwork,
-              model_configs=model_configs,
-          )
-          maxtext_stable_1slice >> maxtext_nightly_1slice
+  run_configs = [
+      f"{LLAMA2_7B}-{BASE_MODE}-{W_BF16_KV_BF16}",
+      f"{LLAMA2_7B}-{BASE_MODE}-{W_INT8_KV_INT8}",
+      f"{LLAMA2_7B}-{CHAT_MODE}-{W_BF16_KV_BF16}",
+      # f"{LLAMA2_7B}-{CHAT_MODE}-{W_INT8_KV_INT8}",
+      f"{LLAMA2_13B}-{BASE_MODE}-{W_BF16_KV_BF16}",
+      f"{LLAMA2_13B}-{BASE_MODE}-{W_INT8_KV_INT8}",
+      f"{LLAMA2_13B}-{CHAT_MODE}-{W_BF16_KV_BF16}",
+      f"{LLAMA2_13B}-{CHAT_MODE}-{W_INT8_KV_INT8}",
+      # f"{LLAMA2_70B}-{CHAT_MODE}-{W_BF16_KV_BF16}",
+      # f"{LLAMA2_70B}-{CHAT_MODE}-{W_INT8_KV_INT8}",
+      # f"{GEMMA_7B}-{BASE_MODE}-{W_BF16_KV_BF16}",
+      # f"{GEMMA_7B}-{BASE_MODE}-{W_INT8_KV_INT8}",
+      # f"{MIXTRAL_8_7B}-{INSTRUCT_MODE}-{W_BF16_KV_BF16}",
+      # f"{MIXTRAL_8_7B}-{INSTRUCT_MODE}-{W_INT8_KV_INT8}",
```

Comment:
The commented out models succeed. I just commented them out to test a subset of them.